libs/shared/src/components/UX/Email/index.tsx

```diff
@@ -21,10 +22,9 @@ export const UxEmail = (props: Props) => {
 
   const { onValidate, tooltip, placeholder, intlLabel, ...rest } = props;
   const addressValidate: ValidateCB = (value) => {
-    const split = value?.indexOf('@');
-    if (split > 0 && (split + 1) < value?.length)
-      return onValidate?.(value) ?? null;
-    return localFormat;
+    return value?.match(validEmail())
+      ? onValidate?.(value) ?? null
+      : localFormat;
   }
```

Comment:
Consistent email validation.

NOTE: this actually accepts email addresses that RBC refuses.  Not keen on duplicating errors into our code because RBC Is too lazy to read the email spec.